### ISSUE TITLE
2025-02-11, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#80]
# Today's Arxiv Papers

(1) [Visual Agentic AI for Spatial Reasoning with a Dynamic API](http://arxiv.org/abs/2502.06787v1), 2025-02-10

(2) [Towards Internet-Scale Training For Agents](http://arxiv.org/abs/2502.06776v1), 2025-02-10

(3) [On the Emergence of Thinking in LLMs I: Searching for the Right Intuition](http://arxiv.org/abs/2502.06773v1), 2025-02-10

(4) [ReasonFlux: Hierarchical LLM Reasoning via Scaling Thought Templates](http://arxiv.org/abs/2502.06772v1), 2025-02-10

(5) [Rationalization Models for Text-to-SQL](http://arxiv.org/abs/2502.06759v1), 2025-02-10

(6) [Gradient Multi-Normalization for Stateless and Scalable LLM Training](http://arxiv.org/abs/2502.06742v1), 2025-02-10

(7) [Resurrecting saturated LLM benchmarks with adversarial encoding](http://arxiv.org/abs/2502.06738v1), 2025-02-10

(8) [VersaPRM: Multi-Domain Process Reward Model via Synthetic Reasoning Data](http://arxiv.org/abs/2502.06737v1), 2025-02-10

(9) [Dynamic Loss-Based Sample Reweighting for Improved Large Language Model Pretraining](http://arxiv.org/abs/2502.06733v1), 2025-02-10

(10) [Can 1B LLM Surpass 405B LLM? Rethinking Compute-Optimal Test-Time Scaling](http://arxiv.org/abs/2502.06703v1), 2025-02-10

(11) [Boosting Self-Efficacy and Performance of Large Language Models via Verbal Efficacy Stimulations](http://arxiv.org/abs/2502.06669v1), 2025-02-10

(12) [Automatic Evaluation of Healthcare LLMs Beyond Question-Answering](http://arxiv.org/abs/2502.06666v1), 2025-02-10

(13) [EfficientLLM: Scalable Pruning-Aware Pretraining for Architecture-Agnostic Edge Language Models](http://arxiv.org/abs/2502.06663v1), 2025-02-10

(14) [Who Taught You That? Tracing Teachers in Model Distillation](http://arxiv.org/abs/2502.06659v1), 2025-02-10

(15) [Unbiased Evaluation of Large Language Models from a Causal Perspective](http://arxiv.org/abs/2502.06655v1), 2025-02-10

(16) [In-Context Learning (and Unlearning) of Length Biases](http://arxiv.org/abs/2502.06653v1), 2025-02-10

(17) [Transparent NLP: Using RAG and LLM Alignment for Privacy Q&A](http://arxiv.org/abs/2502.06652v1), 2025-02-10

(18) [Steel-LLM:From Scratch to Open Source -- A Personal Journey in Building a Chinese-Centric LLM](http://arxiv.org/abs/2502.06635v1), 2025-02-10

(19) [Automatic Annotation Augmentation Boosts Translation between Molecules and Natural Language](http://arxiv.org/abs/2502.06634v1), 2025-02-10

(20) [Combining Large Language Models with Static Analyzers for Code Review Generation](http://arxiv.org/abs/2502.06633v1), 2025-02-10

(21) [Do we really have to filter out random noise in pre-training data for language models?](http://arxiv.org/abs/2502.06604v1), 2025-02-10

(22) [Hephaestus: Improving Fundamental Agent Capabilities of Large Language Models through Continual Pre-Training](http://arxiv.org/abs/2502.06589v1), 2025-02-10

(23) [A Survey on Video Analytics in Cloud-Edge-Terminal Collaborative Systems](http://arxiv.org/abs/2502.06581v1), 2025-02-10

(24) [LawGPT: Knowledge-Guided Data Generation and Its Application to Legal LLM](http://arxiv.org/abs/2502.06572v1), 2025-02-10

(25) [Large Language Models Meet Symbolic Provers for Logical Reasoning Evaluation](http://arxiv.org/abs/2502.06563v1), 2025-02-10

(26) [Position: It's Time to Act on the Risk of Efficient Personalized Text Generation](http://arxiv.org/abs/2502.06560v1), 2025-02-10

(27) [ProjectTest: A Project-level Unit Test Generation Benchmark and Impact of Error Fixing Mechanisms](http://arxiv.org/abs/2502.06556v1), 2025-02-10

(28) [Is API Access to LLMs Useful for Generating Private Synthetic Tabular Data?](http://arxiv.org/abs/2502.06555v1), 2025-02-10

(29) [Efficient Scientific Full Text Classification: The Case of EICAT Impact Assessments](http://arxiv.org/abs/2502.06551v1), 2025-02-10

(30) [Ignore the KL Penalty! Boosting Exploration on Critical Tokens to Enhance RL Fine-Tuning](http://arxiv.org/abs/2502.06533v1), 2025-02-10

(31) [GuideLLM: Exploring LLM-Guided Conversation with Applications in Autobiography Interviewing](http://arxiv.org/abs/2502.06494v1), 2025-02-10

(32) [Recent Advances in Discrete Speech Tokens: A Review](http://arxiv.org/abs/2502.06490v1), 2025-02-10

(33) [Adaptive Prompting: Ad-hoc Prompt Composition for Social Bias Detection](http://arxiv.org/abs/2502.06487v1), 2025-02-10

(34) [UniMoD: Efficient Unified Multimodal Transformers with Mixture-of-Depths](http://arxiv.org/abs/2502.06474v1), 2025-02-10

(35) [KARMA: Leveraging Multi-Agent LLMs for Automated Knowledge Graph Enrichment](http://arxiv.org/abs/2502.06472v1), 2025-02-10

(36) [A Survey of Theory of Mind in Large Language Models: Evaluations, Representations, and Safety Risks](http://arxiv.org/abs/2502.06470v1), 2025-02-10

(37) [MATH-Perturb: Benchmarking LLMs' Math Reasoning Abilities against Hard Perturbations](http://arxiv.org/abs/2502.06453v1), 2025-02-10

(38) [CoS: Chain-of-Shot Prompting for Long Video Understanding](http://arxiv.org/abs/2502.06428v1), 2025-02-10

(39) [Generating Privacy-Preserving Personalized Advice with Zero-Knowledge Proofs and LLMs](http://dx.doi.org/10.1145/3701716.3715597), 2025-02-10

(40) [Occ-LLM: Enhancing Autonomous Driving with Occupancy-Based Large Language Models](http://arxiv.org/abs/2502.06419v1), 2025-02-10

(41) [Systematic Outliers in Large Language Models](http://arxiv.org/abs/2502.06415v1), 2025-02-10

(42) [SynthDetoxM: Modern LLMs are Few-Shot Parallel Detoxification Data Annotators](http://arxiv.org/abs/2502.06394v1), 2025-02-10

(43) [How Humans Help LLMs: Assessing and Incentivizing Human Preference Annotators](http://arxiv.org/abs/2502.06387v1), 2025-02-10

(44) [Simulation as Reality? The Effectiveness of LLM-Generated Data in Open-ended Question Assessment](http://arxiv.org/abs/2502.06371v1), 2025-02-10

(45) [Calibrating LLMs with Information-Theoretic Evidential Deep Learning](http://arxiv.org/abs/2502.06351v1), 2025-02-10

(46) [AiRacleX: Automated Detection of Price Oracle Manipulations via LLM-Driven Knowledge Mining and Prompt Generation](http://arxiv.org/abs/2502.06348v1), 2025-02-10

(47) [Expect the Unexpected: FailSafe Long Context QA for Finance](http://arxiv.org/abs/2502.06329v1), 2025-02-10

(48) [Can AI Examine Novelty of Patents?: Novelty Evaluation Based on the Correspondence between Patent Claim and Prior Art](http://arxiv.org/abs/2502.06316v1), 2025-02-10

(49) [Latent Convergence Modulation in Large Language Models: A Novel Approach to Iterative Contextual Realignment](http://arxiv.org/abs/2502.06302v1), 2025-02-10

(50) [SeaExam and SeaBench: Benchmarking LLMs with Local Multilingual Questions in Southeast Asia](http://arxiv.org/abs/2502.06298v1), 2025-02-10

(51) [Jakiro: Boosting Speculative Decoding with Decoupled Multi-Head via MoE](http://arxiv.org/abs/2502.06282v1), 2025-02-10

(52) [DebateBench: A Challenging Long Context Reasoning Benchmark For Large Language Models](http://arxiv.org/abs/2502.06279v1), 2025-02-10

(53) [Emergent Response Planning in LLM](http://arxiv.org/abs/2502.06258v1), 2025-02-10

(54) [K-ON: Stacking Knowledge On the Head Layer of Large Language Model](http://arxiv.org/abs/2502.06257v1), 2025-02-10

(55) [Find Central Dogma Again](http://arxiv.org/abs/2502.06253v1), 2025-02-10

(56) [Amplifying Minority Voices: AI-Mediated Devil's Advocate System for Inclusive Group Decision-Making](http://dx.doi.org/10.1145/3708557.3716334), 2025-02-10

(57) [Confidence Improves Self-Consistency in LLMs](http://arxiv.org/abs/2502.06233v1), 2025-02-10

(58) [LessLeak-Bench: A First Investigation of Data Leakage in LLMs Across 83 Software Engineering Benchmarks](http://arxiv.org/abs/2502.06215v1), 2025-02-10

(59) [Unveiling the Capabilities of Large Language Models in Detecting Offensive Language with Annotation Disagreement](http://arxiv.org/abs/2502.06207v1), 2025-02-10

(60) [C-3PO: Compact Plug-and-Play Proxy Optimization to Achieve Human-like Retrieval-Augmented Generation](http://arxiv.org/abs/2502.06205v1), 2025-02-10

(61) [Non-literal Understanding of Number Words by Language Models](http://arxiv.org/abs/2502.06204v1), 2025-02-10

(62) [Timing Matters: How Using LLMs at Different Timings Influences Writers' Perceptions and Ideation Outcomes in AI-Assisted Ideation](http://arxiv.org/abs/2502.06197v1), 2025-02-10

(63) [Can LLMs Replace Human Evaluators? An Empirical Study of LLM-as-a-Judge in Software Engineering](http://arxiv.org/abs/2502.06193v1), 2025-02-10

(64) [Uncertainty-Aware Adaptation of Large Language Models for Protein-Protein Interaction Analysis](http://arxiv.org/abs/2502.06173v1), 2025-02-10

(65) [Scaling Public Health Text Annotation: Zero-Shot Learning vs. Crowdsourcing for Improved Efficiency and Labeling Accuracy](http://arxiv.org/abs/2502.06150v1), 2025-02-10

(66) [Optimizing Knowledge Integration in Retrieval-Augmented Generation with Self-Selection](http://arxiv.org/abs/2502.06148v1), 2025-02-10

(67) [LegalViz: Legal Text Visualization by Text To Diagram Generation](http://arxiv.org/abs/2502.06147v1), 2025-02-10

(68) [LCIRC: A Recurrent Compression Approach for Efficient Long-form Context and Query Dependent Modeling in LLMs](http://arxiv.org/abs/2502.06139v1), 2025-02-10

(69) [CSR-Bench: Benchmarking LLM Agents in Deployment of Computer Science Research Repositories](http://arxiv.org/abs/2502.06111v1), 2025-02-10

(70) [RALLRec: Improving Retrieval Augmented Large Language Model Recommendation with Representation Learning](http://arxiv.org/abs/2502.06101v1), 2025-02-10

(71) [ConMeC: A Dataset for Metonymy Resolution with Common Nouns](http://arxiv.org/abs/2502.06087v1), 2025-02-10

(72) [Is a Peeled Apple Still Red? Evaluating LLMs' Ability for Conceptual Combination with Property Type](http://arxiv.org/abs/2502.06086v1), 2025-02-10

(73) [Deconstructing Depression Stigma: Integrating AI-driven Data Collection and Analysis with Causal Knowledge Graphs](http://arxiv.org/abs/2502.06075v1), 2025-02-09

(74) [Benchmarking Prompt Sensitivity in Large Language Models](http://arxiv.org/abs/2502.06065v1), 2025-02-09

(75) [Benchmarking Prompt Engineering Techniques for Secure Code Generation with GPT Models](http://arxiv.org/abs/2502.06039v1), 2025-02-09

(76) [Investigating Compositional Reasoning in Time Series Foundation Models](http://arxiv.org/abs/2502.06037v1), 2025-02-09

(77) [Dual Caption Preference Optimization for Diffusion Models](http://arxiv.org/abs/2502.06023v1), 2025-02-09

(78) [Media Bias Detector: Designing and Implementing a Tool for Real-Time Selection and Framing Bias Analysis in News Coverage](http://dx.doi.org/10.1145/3706598.3713716), 2025-02-09

(79) [Transformers versus the EM Algorithm in Multi-class Clustering](http://arxiv.org/abs/2502.06007v1), 2025-02-09

(80) [Analysis of LLM as a grammatical feature tagger for African American English](http://arxiv.org/abs/2502.06004v1), 2025-02-09

(81) [HamRaz: A Culture-Based Persian Conversation Dataset for Person-Centered Therapy Using LLM Agents](http://arxiv.org/abs/2502.05982v1), 2025-02-09

(82) [$μ$nit Scaling: Simple and Scalable FP8 LLM Training](http://arxiv.org/abs/2502.05967v1), 2025-02-09

(83) [MetaChain: A Fully-Automated and Zero-Code Framework for LLM Agents](http://arxiv.org/abs/2502.05957v1), 2025-02-09

(84) [Cyri: A Conversational AI-based Assistant for Supporting the Human User in Detecting and Responding to Phishing Attacks](http://arxiv.org/abs/2502.05951v1), 2025-02-09

(85) [Acceleration Multiple Heads Decoding for LLM via Dynamic Tree Attention](http://arxiv.org/abs/2502.05947v1), 2025-02-09

(86) ["Let the AI conspiracy begin..." Language Model coordination is just one inference-intervention away](http://arxiv.org/abs/2502.05945v1), 2025-02-09

(87) [Multi-granular Training Strategies for Robust Multi-hop Reasoning Over Noisy and Heterogeneous Knowledge Sources](http://arxiv.org/abs/2502.05944v1), 2025-02-09

(88) [Learning to Substitute Words with Model-based Score Ranking](http://arxiv.org/abs/2502.05933v1), 2025-02-09

(89) [A Generative Framework for Bidirectional Image-Report Understanding in Chest Radiography](http://arxiv.org/abs/2502.05926v1), 2025-02-09

(90) [Can Generative Agent-Based Modeling Replicate the Friendship Paradox in Social Media Simulations?](http://arxiv.org/abs/2502.05919v1), 2025-02-09

(91) [GRAIT: Gradient-Driven Refusal-Aware Instruction Tuning for Effective Hallucination Mitigation](http://arxiv.org/abs/2502.05911v1), 2025-02-09

(92) [EvoAgent: Agent Autonomous Evolution with Continual World Model for Long-Horizon Tasks](http://arxiv.org/abs/2502.05907v1), 2025-02-09

(93) [Enhancing Depression Detection with Chain-of-Thought Prompting: From Emotion to Reasoning Using Large Language Models](http://arxiv.org/abs/2502.05879v1), 2025-02-09

(94) [Retrieval-augmented Large Language Models for Financial Time Series Forecasting](http://arxiv.org/abs/2502.05878v1), 2025-02-09

(95) [Self-Training Large Language Models for Tool-Use Without Demonstrations](http://arxiv.org/abs/2502.05867v1), 2025-02-09

(96) [Fact-or-Fair: A Checklist for Behavioral Testing of AI Models on Fairness-Related Queries](http://arxiv.org/abs/2502.05849v1), 2025-02-09

(97) [Training-free Anomaly Event Detection via LLM-guided Symbolic Pattern Discovery](http://arxiv.org/abs/2502.05843v1), 2025-02-09

(98) [LegalSeg: Unlocking the Structure of Indian Legal Judgments Through Rhetorical Role Classification](http://arxiv.org/abs/2502.05836v1), 2025-02-09

(99) [Delta - Contrastive Decoding Mitigates Text Hallucinations in Large Language Models](http://arxiv.org/abs/2502.05825v1), 2025-02-09

(100) [The Curse of Depth in Large Language Models](http://arxiv.org/abs/2502.05795v1), 2025-02-09
